lib/tizen_build_target.dart

```diff
@@ -162,6 +166,60 @@ class TizenPlugins extends Target {
         sharedLib.copySync(outputDir.childFile(sharedLib.basename).path);
       }
     }
+
+    final Depfile pluginDepfile = collectPluginSources(
+      nativePlugins,
+      environment,
+      ephemeralDir,
+    );
+    final DepfileService depfileService = DepfileService(
+      fileSystem: environment.fileSystem,
+      logger: environment.logger,
+    );
+    depfileService.writeToFile(
+      pluginDepfile,
+      environment.buildDir.childFile('tizen_plugins.d'),
+    );
+  }
+
+  Depfile collectPluginSources(
+    List<TizenPlugin> nativePlugins,
+    Environment environment,
+    Directory ephemeralDir,
```

Comment:
When you write a new function, please try to make it readable (easy to understand what the function does quickly from its signature) by using descriptive function name and parameter names.

Reply:
How about this?
```dart
Depfile createDepfile({
    @required List<TizenPlugin> nativePlugins,
    @required Directory compiledPluginsDir,
  })
```

Reply:
Looks better. You may also make the function private.